playwright-e2e/tests/dsm/osteo/os1-participant-pvavgt.spec.ts

```diff
@@ -0,0 +1,131 @@
+import { Page, expect } from '@playwright/test';
+import { test } from 'fixtures/dsm-fixture';
+import { StudyEnum } from 'dsm/component/navigation/enums/selectStudyNav-enum';
+import ParticipantListPage from 'dsm/pages/participant-list-page';
+import { AdditionalFilter, CustomViewColumns } from 'dsm/component/filters/sections/search/search-enums';
+import { TabEnum } from 'dsm/component/tabs/enums/tab-enum';
+import MedicalRecordsTab from 'dsm/pages/medical-records/medical-records-tab';
+import OncHistoryTab from 'dsm/component/tabs/onc-history-tab';
+import CohortTag from 'dsm/component/cohort-tag';
+import { SamplesNavEnum } from 'dsm/component/navigation/enums/samplesNav-enum';
+import SearchPage, { SearchByField } from 'dsm/pages/samples/search-page';
+import { Navigation } from 'dsm/component/navigation/navigation';
+import { SortOrder } from 'dss/component/table';
+import { WelcomePage } from 'dsm/pages/welcome-page';
+
+const { DSM_BASE_URL } = process.env;
+
+/**
+  Run on Test env only!
+  Participant Short ID is hard-coded. Participant data is expected to be unchanged.
+*/
+
+test.describe.serial('Osteo1 Participant', () => {
+  test.skip(DSM_BASE_URL === undefined || (DSM_BASE_URL as string).indexOf('test') === -1);
+
+  const shortID = 'PVAVGT';
+  const osteo1 = StudyEnum.OSTEO;
+
+  test(`Should find by Cohort Tag`, async ({page, request}) => {
+    const participantListPage: ParticipantListPage = await ParticipantListPage.goto(page, osteo1, request);
+
+    const tagNameColumn = 'Cohort Tag Name';
+    const customizeViewPanel = participantListPage.filters.customizeViewPanel;
+    await customizeViewPanel.open();
+    await customizeViewPanel.selectColumns(CustomViewColumns.COHORT_TAGS, [tagNameColumn]);
+
+    const searchPanel = participantListPage.filters.searchPanel;
+    const participantListTable = participantListPage.participantListTable;
+
+    // Should not find any Osteo2 participant when search by Osteo2 Cohort tag: "OS PE-CGS"
```

Comment:
just a heads up - I think for OS1 this search might be borked somehow or setup differently compared to OS2 - it actually should return pts that have the tag 'OS PE-CGS' since then study admin will be able to tell from a glance which pts are in both studies (regardless of looking in OS1 or OS2) - I went to manually check it and while cohort tag search seems to be working great in OS2; it's weird in OS1 - searching for 'OS' tag seems to work as expected - searching for a different tag e.g. 'Nov 7', 'Hi' [at least 1 pt has this], does not return anything - got a ticket for it here: https://broadworkbench.atlassian.net/browse/PEPPER-1293

Reply:
So, should I remove this check (search) from test? since it suppose to return some participants in the Osteo study.

Reply:
Removed search by OS2 cohort tag from the test.

Reply:
sorry just saw this - and yeah, sounds good